plugins/wazuh-core/package.json

```diff
@@ -15,18 +15,35 @@
     "test:server": "plugin-helpers test:server",
     "test:browser": "plugin-helpers test:browser",
     "test:jest": "node scripts/jest --runInBand",
-    "test:jest:runner": "node scripts/runner test"
+    "test:jest:runner": "node scripts/runner test",
+    "lint": "eslint \"{public,server,common}/**/*.{js,jsx,ts,tsx,json}\"",
+    "knip": "knip --files"
   },
   "dependencies": {
     "axios": "^1.6.1",
     "json2csv": "^4.1.2",
     "jwt-decode": "^3.1.2",
     "md5": "^2.3.0",
-    "node-cron": "^3.0.2"
+    "node-cron": "^3.0.2",
+    "typescript": "^5.5.2"
```

Comment:
**nitpick**: this should be a dev dependency.

Reply:
Done